config.sample.yaml

```diff
@@ -168,6 +168,9 @@ thumbnails:
   # thumbnails, set this to false. If disabled, regular thumbnails will be returned.
   allowAnimated: true
 
+  # Force animated thumbnails, if available
+  forceAnimated: true
```

Comment:
This should probably be `defaultAnimated` instead of `forceAnimated`. Losing the ability to say `&animated=false` isn't desirable.

Reply:
I am unsure what you mean with the suggested behaviour, would you mind to elaborate?